frontend/src/components/ScheduleSearch.tsx

```diff
@@ -46,9 +58,10 @@ const CourseCombobox = ({
   }, [activeSchedule]);
 
   function getCourses() {
-    return allCourses.filter(
+    return allCoursesSearchable.filter(
       (course) =>
         (course.courseID.includes(inputValue) ||
+          course.courseIDDehyphenated.includes(inputValue) ||
           course.name.toLowerCase().includes(inputValue)) &&
         selectedItems.map(({ courseID }) => courseID).indexOf(course.courseID) <
```

Comment:
This works, but it would be faster to hyphenate inputValue if necessary and then check only on inputValue.